devito/ir/iet/nodes.py

```diff
@@ -60,6 +60,12 @@ class Node(Signer):
     appears in this list are treated as traversable fields.
     """
 
+    _ccode_handler = None
+    """
+    Customizable by subclasses, in particular Operator subclasses which define
+    backend-specific nodes and, su such, require node-specific handlers.
```

Comment:
typo

Reply:
noted, will fix

Reply:
... in my upcoming PR, if you don't mind, as apparently no other changes are requested here given yours and @mloubout 's comments

Reply:
ok

Reply:
fixed now as I had to push anyway